ion_phys/common.py

```diff
@@ -451,7 +451,7 @@ def calc_M1(self):
         eyeI = np.identity(I_dim)
 
         for level, data in self.levels.items():
-            lev = level.slice()
+            lev = data.slice()
```

Comment:
Thanks for fixing. The variable names are all over the place here (`lev`, `level`, `data`) and make the code somewhat hard to follow (my fault IIRC). I'd happily accept a PR at some point that made this a bit more civilized if you can think of better terminology...

Reply:
Yes I agree, it was a bit hard to try and work out what was going on to start with. Another slightly confusing thing is the multiple definitions of the slice() function, of which one is native to python.

Reply:
Yes, I wasn't totally happy with `slice` function name among a few others (e.g. `delta`) and would be very open to suggestions for better replacements.